example/model/openai_jupyter_notebook_QA.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
If this notebook is to split markdown and generate QA, consider rename it to `openai_markdown_split_and_qa_gen.ipynb`?

Reply:
Splitting markdown is only partial of this notebook's purpose. We can refactor the function to be a flow in the future.